common/djangoapps/util/tests/test_submit_feedback.py

```diff
@@ -19,6 +23,7 @@
 TEST_SUPPORT_EMAIL = "support@example.com"
 TEST_ZENDESK_CUSTOM_FIELD_CONFIG = {"course_id": 1234, "enrollment_mode": 5678}
 TEST_REQUEST_HEADERS = {
+    "ENTERPRISE_CUSTOMER_NAME": "TestShib",
```

Comment:
I'm going to go out on a limb here and say that if you're including a custom variable inline with a list of standard semantic HTTP headers for testing purposes, you're doing testing wrong. This should not be here. I can't tell you where it should be (if anywhere) without spending a lot of time on it, but it should not go here.

I suspect that if you set up Enterprise as a custom field instead of appending it to the body (as discussed above), you'll have a much better time.